README.md

```diff
@@ -6,18 +6,26 @@ Install
 
 Instructions based on Ubuntu 14.04.
 
-Install the system package dependencies & virtualenv:
+If you're installing to your local Ubuntu 14.04:
 
 ```
-$ sudo apt-get install `cat debian_packages.lst`
-$ pip3 install --user virtualenv && pip3 install --user virtualenvwrapper
+$ make install_system_dependencies
 ```
 
-Ensure you load virtualenv with Python 3 in `~/.bashrc`:
```

Comment:
@brousch You need to keep mentioning the need to add this to `.bashrc` for local (non-Vagrant) installs, this information is currently lost with your changes.

Also, for Vagrant, why not go all the way, and do this change in `.bashrc` on the Vagrant VM automatically, rather thatn asking the user to go do it manually himself?